### ISSUE TITLE
learning rate cycling

### ISSUE THREAD

**Issue by GotG** [zylo117/Yet-Another-EfficientDet-Pytorch#231]
how can we use a learning rate cycling scheduler to train EffDet? has anyone tried it?
